R/Connection.R

```diff
@@ -55,7 +55,7 @@ OdbcConnection <- function(
   stopifnot(all(has_names(attributes)))
   stopifnot(all(names(attributes) %in% SUPPORTED_CONNECTION_ATTRIBUTES))
 
-  connection_string <- paste0(.connection_string, paste(collapse = ";", sep = "=", names(args), args))
+  connection_string <- paste0(.connection_string, connection_string(args))
```

Comment:
```suggestion
  connection_string <- paste0(.connection_string, build_connection_string(args))
```

Will spend some time to make sure I've vetted the new helper well, but first thing: